Dockerfile

```diff
@@ -15,6 +15,9 @@ RUN command -v yarn || npm i -g yarn
 
 USER root
 
+## Upgrade packages
+RUN dnf update -y --setopt=install_weak_deps=0 && rm -rf /var/cache/yum
+
```

Comment:
Only the upgrade of the packages of the base container image nginx-122 is required.